adsft/extraction.py

```diff
@@ -618,6 +630,13 @@ def extract_multi_content(self, translate=False, decode=False):
                                  )
                     raise Exception(traceback.format_exc())
 
+                if META_CONTENT[self.meta_name][content_name]['type'] == 'string':
+                    meta_out[content_name] = "\n".join(all_text_content)
+                elif len(all_text_content) > 0:
+                    meta_out[content_name] = all_text_content[0]
+                else:
+                    meta_out[content_name] = all_text_content

```

Comment:
Before, `meta_out[content_name]` was only assigned if `text_content` was extracted, but this last `else` seems like when we do not extract anything, we assign a empty list to `meta_out[content_name]`, no?

Reply:
You are correct, I have taken out the 'else' statement to stop this from happening.